internal/config/config.go

```diff
@@ -64,6 +65,27 @@ type ConfigCluster struct {
 	Location string `yaml:"location"`
 }
 
+type ConfigInput struct {
+	GKEApi        GKEApiInput   `yaml:"gke-api"`
```

Comment:
other params in yaml file we were naming in camel case - shouldn't we follow that convention in this case also? (eg. `clientTimeoutSeconds`)

Reply:
Good point. The only reason why I proposed `gke-api` instead of `gkeAPI` is alignment with an input identifier.
I wanted things to be consistent, so :
* user configures input by it's ID
* the same input ID is used in policies

What do you think about changing input identifier to "gkeAPI" then ? 
(and other input ids correspondingly)

Reply:
good idea!

Reply:
I've changed the IDs to follow camel case then